### PR TITLE
Fix assets pathing, and add fonts to document tests

### PR DESCRIPTION
**Pull request by bendik**
is this a 🐛 bug fix

## Checklist
- [x] tests pass

## Context
https://github.com/choojs/bankai/pull/453

## Semver Changes
Patch

### CODE REVIEW

File: lib/graph-assets.js [hunk 1]
```diff
@@ -35,7 +35,9 @@ function node (state, createEdge) {
   })
 
   tracker.on('change', function () {
-    var list = tracker.list()
+    var list = tracker.list().map(function (file) {
+      return file.replace(basedir, '')
```
Comment (goto-bus-stop) [inline lib/graph-assets.js]: would using the `path.relative` function instead also work?

**Comment by bendik** [comment]
Don't know why the tests fail here.
Tried running it with same node version as CI, and it still passes on my end. And I can't quite make out what warrants the output i see on CI.
Any ideas?

**Comment by bendik** [comment]
Seems to be a POSIX/windows thing with path relative https://devdocs.io/node/path#path_path_relative_from_to

Fix incoming.

**Comment by goto-bus-stop** [comment]
:package: 9.10.4. thanks!

**Pull request merged**